lighthouse/lighthouserc.js

```diff
@@ -1,7 +1,6 @@
 const AUDIT_URLS = {
   FRONT_PAGE: {
     live: ['https://www.bbc.com/serbian/cyr'],
-    test: ['https://www.test.bbc.com/serbian/cyr?renderer_env=live'],
```

Comment:
I need to check if the lighthouse tests can actually run OK without a configuration for `test`? @holchris Let's have a quick call so I can show you how